api/storage_service.cc

```diff
@@ -992,12 +992,12 @@ void set_snapshot(http_context& ctx, routes& r) {
 
     ss::take_snapshot.set(r, [](std::unique_ptr<request> req) {
         auto tag = req->get_query_param("tag");
-        auto column_family = req->get_query_param("cf");
+        auto column_families = split(req->get_query_param("cf"), ",");
 
```

Comment:
We should have a separate REST endpoint that takes a list.

Reply:
In the new version of the API, definitely, but this is in sync with how the API looks like today for all other column_family commands and it was a request to keep the naming convention

Reply:
ok (yuch)